test/tests/neml_regression/neml_regression.i

```diff
@@ -20,16 +18,19 @@
 
 [GlobalParams]
   displacements = 'disp_x disp_y disp_z'
-  data_file = 'gold/test1_out.csv'
 []
 
 [Mesh]
-  type = GeneratedMesh
-  dim = 3
+  [cube]
+    type = GeneratedMeshGenerator
+    dim = 3
+  []
 []
 
 [Modules/TensorMechanics/Master]
   [all]
+    strain = SMALL
+    incremental = true
```

Comment:
Why incremental?

Reply:
I think those are both the defaults, actually. It runs the same without them being set.

Reply:
`SMALL` is the defsult. If not supplied `incremental` is set to `false` for small strain. You are selecting an incremental small strain formulation, while we usually use total small strain.

Reply:
```
  // determine if incremental strains are to be used
  if (isParamValid("incremental"))
  {
    const bool incremental = getParam<bool>("incremental");
    if (!incremental && _strain == Strain::Small)
      _strain_and_increment = StrainAndIncrement::SmallTotal;
    else if (!incremental && _strain == Strain::Finite)
      _strain_and_increment = StrainAndIncrement::FiniteTotal;
    else if (incremental && _strain == Strain::Small)
      _strain_and_increment = StrainAndIncrement::SmallIncremental;
    else if (incremental && _strain == Strain::Finite)
      _strain_and_increment = StrainAndIncrement::FiniteIncremental;
    else
      mooseError("Internal error");
  }
  else
  {
    if (_strain == Strain::Small)
      _strain_and_increment = StrainAndIncrement::SmallTotal;  <------------------ DEFAULT FOR SMALL
    else if (_strain == Strain::Finite)
      _strain_and_increment = StrainAndIncrement::FiniteIncremental;
    else
      mooseError("Internal error");
  }
```

Reply:
Well, now I'm puzzled. I would have thought that these would require incremental.

Reply:
Well, looks like the NEML stress calculator actually only uses the total strain, so it doesn't make any difference how that is set.